.travis.yml

```diff
@@ -1,27 +1,31 @@
 language: python
 
+branches:
+  except:
+    - gh-pages
+
+cache:
+  directories:
+    - "${HOME}/virtualenv"
+    - "${TRAVIS_BUILD_DIR}/.tox"
+
 python:
   - 2.6
   - 2.7
   - 3.3
   - 3.4
+  - 3.5
+  - 3.6
```

Comment:
Extension is not tested for python 3.5 and 3.6 environments and has failed build on Travis. Allow failures for python 3.5 as well.